### PR TITLE
Add nubis-puppet-eip module to Puppetfile

### PR DESCRIPTION
**Pull request by limed**
This resolves issue #164

### CODE REVIEW

File: nubis/Puppetfile [hunk 1]
```diff
@@ -59,6 +59,9 @@ mod 'nubis/nubis_configuration',
 mod 'nubis/nubis_storage',
     :git => 'https://github.com/Nubisproject/nubis-puppet-storage.git'
 
+mod 'nubis/nubis_eip',
+    :git => 'https://github.com/Nubisproject/nubis-puppet-eip.git'
+
```
Comment (tinnightcap) [inline nubis/Puppetfile]: If you expect this to be stable you should add a signed tag (release) on github and add
```
:ref => '0.0.1'
```
here to pin to a version number. Feel free to ping me on IRC for any help with the [release process](https://github.com/Nubisproject/nubis-docs/blob/master/RELEASING.md#process).

**Comment by gozer** [comment]
Missing a release number / tag to pin to

**Comment by limed** [comment]
Pinned to version 0.0.1

**Comment by tinnightcap** [comment]
r=jd

**Pull request merged**